x-pack/plugins/security_solution/server/endpoint/routes/metadata/support/test_support.ts

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
suggestion: name this file `mocks`. Seems to be common across kibana. Also, I often see the function names have the word `mock` in it - ex. `createV1SearchResponseMock()`